tests/lsp_features/completion.zig

```diff
@@ -1253,6 +1253,45 @@ test "completion - usingnamespace" {
         .{ .label = "alpha", .kind = .Function, .detail = "fn alpha() void" },
         .{ .label = "beta", .kind = .Function, .detail = "fn beta() void" },
     });
+    try testCompletion(
+        \\pub const chip_mod = struct {
+        \\    pub const types = struct {
+        \\        // type definitions for peripherals here
+        \\    };
+        \\    pub const devices = struct {
+        \\        pub const chip1 = struct {
+        \\            chip1fld1: bool,
+        \\            chip1fld2: u32,
+        \\            // peripherals and interrupt table here ...
+        \\            pub const peripherals = struct {
+        \\                pub const IDT = 1;
+        \\            };
+        \\            pub fn chip1fn1() void {}
+        \\            pub fn chip1fn2(_: u32) void {}
+        \\        };
+        \\        pub const chip2 = struct {
+        \\            chip2fld1: u64,
+        \\            chip2ffld2: u32,
+        \\            // peripherals and interrupt table here ...
+        \\            pub fn chip2fn1() void {}
+        \\        };
+        \\    };
+        \\};
+        \\const chip = struct {
+        \\    const inner = chip_mod; //@import("chip");
+        \\    pub const types = inner.types;
+        \\    pub usingnamespace @field(inner.devices, "chip1");
+        \\};
+        \\test {
+        \\    _ = chip.<cursor>;
+        \\}
+    , &.{
+        .{ .label = "inner", .kind = .Constant, .detail = "const chip_mod = struct" },
+        .{ .label = "types", .kind = .Constant, .detail = "const types = struct" },
+        .{ .label = "peripherals", .kind = .Constant, .detail = "const peripherals = struct" },
+        .{ .label = "chip1fn1", .kind = .Function, .detail = "fn chip1fn1() void" },
+        .{ .label = "chip1fn2", .kind = .Function, .detail = "fn chip1fn2(_: u32) void" },
+    });
```

Comment:
could this test case be made slightly smaller, maybe also add a test just for `@field`.